File: week-02/day-02/reverse_list.py
# - Create a variable named `aj`
#   with the following content: `[3, 4, 5, 6, 7]`
# - Reverse the order of the elements in `aj`
# - Print the elements of the reversed `aj`
aj = [3, 4, 5, 6, 7]
for i in range(0,int((len(aj)/2))):
    aj[0+i],aj[-1-i] = aj[-1-i],aj[0+i]
print(aj)

# aj = list(reversed(aj))

File: week-08/day-02/Functions/reverse_list/reverse_list.js
'use strict';
// - Create a variable named `aj`
//   with the following content: `[3, 4, 5, 6, 7]`
// - Reverse the order of the elements in `aj`
// 		- do it with the built in method
//		- do it with creating a new temp array and a loop
// - Print the elements of the reversed `aj`

var aj = [3, 4, 5, 6, 7];
aj.reverse();


var ajk = [3, 4, 5, 6, 7];
for(var i =ajk.length+1; i>=0;i--){
    console.log(ajk[i])
}
console.log(aj)